e2e_samples/dataset_versioning/README.md

```diff
@@ -38,6 +38,28 @@ We refer to [LendingClub issued Loans](https://www.kaggle.com/husainsb/lendingcl
         1. Go to provisioned Azure Data Factory, then click [Author & Monitor] button to open development portal.
         1. Click pencil button at left pane, then select [DeltaCopyPipeline].
         1. Cick [Debug] button to run pipeline.
-    1. **[Need update]** Use Databricks to query Delta versions
+    1. Use Databricks to query Delta versions
+        1. Go to Azure Databricks and create a cluster. Remember to tick the "enable credential passthrough" check under the advanced settings if you plan to use the credential passthrough feature. Make sure you have access to Deltalake either with a RBAC role or Access Control List. [detailed steps](./databricks/README.md)
+        1. Create a notebook in Azure Databricks. Select your cluster and execute these scripts below:
+            ```
```

Comment:
Should add `python` after "```"

```python
# Confirm if you can access deltalake. Update the path with the name of your storage account.
file_path = "abfss://datalake@{your_storage_account_name}.dfs.core.windows.net/lc_loan"
dbutils.fs.ls(file_path)
```

Reply:
thanks :) added